tmcprototype/CentralNode/CentralNode/CONST.py

```diff
@@ -41,6 +42,10 @@
 STR_FALSE = "False"
 STR_DISH_DUPLICATE = "List of the dishes that are already allocated: "
 STR_ASSIGN_RESOURCES_SUCCESS = "Resources are assigned successfully"
+STR_FALSE_TAG = "Resources cannot be released since ReleaseALL Tag is False"
```

Comment:
Consider rewording the message to "False in ReleaseALL tag is not yet supported."

Reply:
Done.

Reply:
Th commit id: [485e1cec5f7608502b3bafde2c932940c3532cfb](https://github.com/ska-telescope/tmc-prototype/pull/21/commits/485e1cec5f7608502b3bafde2c932940c3532cfb)